server/routes/questions/router.test.ts

```diff
@@ -467,6 +515,113 @@ describe(`Submitting questions' responses`, () => {
       },
     ]
 
+    const questionsResponse: Question[] = [
```

Comment:
technically, `responseDate` in all responses in this list is null

Reply:
Fair enough. I'll change

(I was more focused on the overarching flow of questions/answer rather then the comments/date which don't make much of a difference in this context...so there was a fair amount of copy and paste)

Reply:
Changed test data to have `null` dates when they're not required in this commit: https://github.com/ministryofjustice/hmpps-incident-reporting/pull/301/commits/e2c1965538098f906eb63c448ba85c00e6adda0b